packages/app/envs/index.native.ts

```diff
@@ -0,0 +1,14 @@
+import Constants from 'expo-constants';
```

Comment:
can we handle this with the packages/config that we have ?

Reply:
We should then have to create the `.env` file with all variable prefixed by `EXPO_PUBLIC_`.

Reply:
Here is the list of update I plan to do:
- [ ] update imports from `@env` or `app/env` to `@packrat/config`
- [ ] update the workflow `android-preview-build-local.yml` to defined these variables
- [ ] update the readme

Reply:
@Tadjaur see the recent changes to the github action, can we map over these to set the env variables or will we have to call export manually for each one?